File: 🟢 Easy/3232 - Find if Digit Game Can Be Won/_3232FindifDigitGameCanBeWon.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        aSum = 0
        bSum = 0
        for i in range(0, len(nums)):
            div = nums[i] // 10
            if div == 0:
                aSum += nums[i]
            else:
                bSum += nums[i]

        if aSum != bSum:
            return True

        return False

File: 🟢 Easy/3232 - Find if Digit Game Can Be Won/_3232FindifDigitGameCanBeWon.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int aSum = 0;
        int bSum = 0;
        for (int i = 0; i < nums.size(); i++) {
            int div = nums[i] / 10;
            if (div == 0) {
                aSum += nums[i];
            } else {
                bSum += nums[i];
            }
        }

        if (aSum != bSum) {
            return true;
        }
        return false;
    }
};